atlp-rwanda/atlp-pulse-fn
# DP-109 feat: Create rating system Integration

admin is able to query all rating sytems

# PR Description
This PR is used to integrate the page accessible to an admin where he/she will be able to create rating(grading) systems.
# Description of tasks that were expected to be completed

- [x] Admin is be able to query all rating systems 
- [x] Admin is able to create a rating system

# How has this been tested?

- Sign in as an admin 
- Navigate on grading system using the navbar
- Query rating systems using a dropdown
- Click on custom to add new rating system
- Use a dropdown to check if the addded rating system is there


-Screenshots (If appropriate)

Refer to this image to know how grading system are added.

![Screenshot (188)](https://user-images.githubusercontent.com/88627627/190957524-1fe2acd2-efc7-422c-a4a4-28659cd12b3d.png)
